server/src/main/java/online/dipa/hub/services/ProjectService.java

```diff
@@ -45,13 +45,19 @@
     @Autowired
     private ConversionService conversionService;
 
+    @Autowired
+    private UserInformationService userInformationService;
+
     private final ProjectProjectEntityMapper projectMapper = Mappers.getMapper(ProjectProjectEntityMapper.class);
     private final ProjectTaskProjectTaskEntityMapper projectTaskMapper = Mappers.getMapper(ProjectTaskProjectTaskEntityMapper.class);
 
 
     public Project getProjectData(final Long projectId) {
 
+        List<String> userGroups = userInformationService.getUserData().getGroups();
+
         return projectRespository.findAll().stream().map(p -> conversionService.convert(p, Project.class))
+        .filter(t -> userGroups.contains(String.valueOf(t.getId())))
```

Comment:
die Rechte sollten auch in den anderen Methoden überprüft werden wo Projektdaten zurückgegeben oder verändert werden können, besonders natürlich bei den Update 
`public void updateProjectData(final Long projectId, final Project project) {`
`public void updateProjectTask (final Long projectId, final ProjectTask projectTask) {`